pkg/controllers/clusters/clusterstatus/clusterstatus_controller.go

```diff
@@ -53,6 +64,8 @@ func NewController(apiserverURL, parentAPIServerURL string, kubeClient kubernete
 		appPusherEnabled: utilfeature.DefaultFeatureGate.Enabled(features.AppPusher),
 		useSocket:        utilfeature.DefaultFeatureGate.Enabled(features.SocketConnection),
 		parentAPIServer:  parentAPIServerURL,
+		nodeLister:       nodeLister,
```

Comment:
make nodes and pods are synced.

```go
informerFactory.Core().V1().Nodes().Informer().HasSynced
```

```go
informerFactory.Core().V1().Pods().Informer().HasSynced
```

And making sure caches are synced successfully in `Run()` before collecting statistics.

Reply:
oh, my fault